packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -54,50 +51,12 @@ export type SharedTreeBranchChange<TChange> =
 			removedCommits: readonly [GraphCommit<TChange>, ...GraphCommit<TChange>[]];
 	  }
 	| {
-			type: "replace";
+			type: "rebase";
 			change: TaggedChange<TChange> | undefined;
 			removedCommits: readonly GraphCommit<TChange>[];
 			newCommits: readonly GraphCommit<TChange>[];
 	  };
 
-/**
- * Returns the operation that caused the given {@link SharedTreeBranchChange}.
- */
-export function getChangeReplaceType(
```

Comment:
It was fun while it lasted.


...right? ...anyone agree?

Reply:
So glad to see this go away.